### DISCUSSION TITLE
記事の投稿機能追加要望

### DISCUSSION THREAD

**Discussion by hyamashita00** [increments/qiita-discussions#911] [要望・アイデア・不具合]
投稿方法についてですが、日時の投稿予約の機能の追加はお願いすることは可能でしょうか。

**Comment by tanaka-asami415** [comment]
@hyamashita00 
リクエストありがとうございます🙏
予約投稿については複数ご意見もいただいており、有料プラン等でのご提供になるかもしれませんが、将来的に検討したいと思っています！
